cellprofiler/modules/maskobjects.py

```diff
@@ -20,6 +20,13 @@
 partially or fully outside of the region, or retain the whole object
 unless it is fully outside of the region.
 
+*Note on saving images:* You can pass the modified objects along to the
```

Comment:
Maybe "masked objects" instead of "modified objects"? Or, does that sound too much like you'd be converting the objects that were masked (i.e., not visible) instead of the unmasked objects?